docs/gcloud-auth.rst

```diff
@@ -74,14 +127,30 @@ like to use.
 
 .. tip::
 
-    Unless you have an explicit reason to use a PKCS12 key for your
+    Unless you have a specific reason to use a PKCS12/P12 key for your
     service account, we recommend using a JSON key.
 
-Finally, if you are **familiar** with the `oauth2client`_ library, you can
-create a ``credentials`` object and pass it directly:
+User Accounts (3-legged OAuth 2.0) with refresh token
+-----------------------------------------------------
 
-.. code:: python
+The majority of cases are intended to authenticate machines or
+workers rather than actual user accounts. However, it's also
+possible to call Google Cloud APIs with a user account via
+`OAuth 2.0`_.
 
-    client = Client(credentials=credentials)
+.. _OAuth 2.0: https://developers.google.com/identity/protocols/OAuth2
 
-.. _oauth2client: http://oauth2client.readthedocs.org/en/latest/
+.. tip::
+
+    A production application should **use a service account**, but you
+    may wish to use your own personal user account when first getting
+    started with the ``gcloud-python`` library.
+
+This is only supported via Application Default Credentials using
+``gcloud auth login`` as mentioned above. To create these
+credentials directly:
+
+.. code:: python
+
+    from oauth2client.client import GoogleCredentials
+    credentials = GoogleCredentials.get_application_default()
```

Comment:
Gotcha -- I read that as "if you want to use user credentials, use Application Default Credentials which will use gcloud auth login" -- which is different from "you can't force the code to use user credentials... the best you can do is try to use the defaults, and hope that your user credentials are in the list..."